pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -128,3 +113,16 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 
 	return wc, nil
 }
+
+// calculateCPUMaximum calculates the maximum CPU given a limit and a number of cpus while ensuring it's in range [1,10000].
+func calculateCPUMaximum(cpuLimit *resource.Quantity, cpuCount int64) int64 {
+	cpuMaximum := 10 * cpuLimit.MilliValue() / cpuCount
```

Comment:
I am pretty sure this is equivalent to prior code, based on the [order of operations](https://go.dev/ref/spec#Operator_precedence) in golang though we've definitely had trouble in the area of the code base in the past so I feel paranoid 😨. Did you add the tests prior to making the code change to ensure things are the same?  

The code that is linked in the comment does it differently (https://github.com/moby/moby/blob/10866714412aea1bb587d1ad14b2ce1ba4cf4308/daemon/oci_windows.go#L426)  not sure I know why it was done slightly differently here.

@marosset want to double check this simplification is ok?

Reply:
going to resolve this - let me know if you still have concerns.